redisinsight/api/src/modules/database/exeptions/database-already-exists.exception.ts

```diff
@@ -0,0 +1,19 @@
+import { HttpException, HttpExceptionOptions, HttpStatus } from '@nestjs/common';
+import { CustomErrorCodes } from 'src/constants';
+import ERROR_MESSAGES from 'src/constants/error-messages';
+
+export class DatabaseAlreadyExistsException extends HttpException {
+  constructor(databaseId: string, message = ERROR_MESSAGES.DATABASE_ALREADY_EXISTS, options?: HttpExceptionOptions) {
+    const response = {
+      message,
+      statusCode: HttpStatus.CONFLICT,
+      error: 'DatabaseAlreadyExists',
+      errorCode: CustomErrorCodes.DatabaseAlreadyExists,
+      result: {
```

Comment:
let's use "resource" instead of "result" if possible, to be compatible with cloud jobs

Reply:
changed